README.md

```diff
@@ -83,15 +83,16 @@ ones controlled by your test.
 
 We provide a top-level method called `td.replace()` that operates in two
 different modes: CommonJS module replacement and object-property replacement.
-Both modes will, by default, perform a deep clone the real dependency which
-replaces all functions it encounters with fake test double functions that can be
-configured by your test to either stub responses or assert invocations.
+Both modes will, by default, perform a deep clone of the real dependency,
+replacing all functions it encounters with fake test double functions. These
+fake test double functions can be configured by your test to either stub
+responses or assert invocations.
```

Comment:
I reworded this paragraph a little bit to make it flow better (to my mind) and break it up some. I'm not married to my changes here though.